src/intl/en/page-upgrades-vision.json

```
@@ -21,34 +23,34 @@
   "page-upgrades-vision-problems": "Today's problems",
   "page-upgrades-vision-scalability": "Scalability",
   "page-upgrades-vision-scalability-desc": "Ethereum needs to be able to handle more transactions per second without increasing the size of the nodes in the network. Nodes are vital network participants who store and run the blockchain. Increasing node size isn't practical because only those with powerful and expensive computers could do it. To scale, Ethereum needs more transactions per second, coupled with more nodes. More nodes means more security.",
-  "page-upgrades-vision-scalability-desc-3": "The shard chains upgrade will spread the load of the network into 64 new chains. This will give Ethereum room to breathe by reducing congestion and improving speeds beyond the current 15-45 transactions per second limit.",
-  "page-upgrades-vision-scalability-desc-4": "And even though there will be more chains, this will actually require less work from validators - the maintainers of the network. Validators will only need to 'run' their shard and not the entire Ethereum chain. This makes nodes more lightweight, allowing Ethereum to scale and remain decentralized.",
+  "page-upgrades-vision-scalability-desc-3": "Sharding upgrades will spread the data storage requirements across the entire network, no longer requiring every node to hold 100% of the data. Although this doesn't directly address scaling the execution of transactions, this problem is being addressed directly by layer 2 rollup solutions.",
+  "page-upgrades-vision-scalability-desc-4": "Rollups need cheap storage on layer 1 though to be most effective. Data sharding will give Ethereum room to breathe by maximizing the efficiency on rollups, enabling exponential improvements beyond the current 15-45 transactions per second limit. Validators will only need to sample the data to make sure it all has been made available.",
```

Don't think the last sentence is relevant if we're just trying to explain that sharding makes rollups cheaper.

```suggestion
  "page-upgrades-vision-scalability-desc-4": "Rollups need cheap storage on layer 1 though to be most effective. Data sharding will give Ethereum room to breathe by maximizing the efficiency on rollups, enabling exponential improvements beyond the current 15-45 transactions per second limit.",
```

---

DAS is how/why they won't have to store all of the data in order to make it cheaper
```suggestion
  "page-upgrades-vision-scalability-desc-4": "Rollups need cheap storage on layer 1 though to be most effective. Data sharding will give Ethereum room to breathe by maximizing the efficiency on rollups, enabling exponential improvements beyond the current 15-45 transactions per second limit.",
```